passenger-service/src/main/java/com/me/passengerservice/service/PassengerService.java

```diff
@@ -0,0 +1,13 @@
+package com.me.passengerservice.service;
+
+import com.me.passengerservice.dto.request.PassengerRequest;
+import com.me.passengerservice.dto.response.PassengerListResponse;
+import com.me.passengerservice.dto.response.PassengerResponse;
+
+public interface PassengerService {
```

Comment:
Minor: If you have a small application and no hierarchy for service, it it better not to create redundant interfaces for each service implementation